index.js

```diff
@@ -426,18 +435,10 @@ function Argv (processArgs, cwd) {
       }
     }
 
+    // we must run completions first, a user might
+    // want to complete the --help or --version option.
     Object.keys(argv).forEach(function (key) {
```

Comment:
no, it's because I cargo-culted the code from inside the loop that did check for multiple cases. I like your snipped better, and will update the pull.